hw/bitstream/BUILD

```diff
@@ -57,17 +59,44 @@ filegroup(
     name = "mask_rom",
     srcs = select({
         "bitstream_skip": ["skip.bit"],
-        "bitstream_vivado": ["//hw/bitstream/vivado:fpga_cw310_mask_rom"],
+        "bitstream_vivado": [":fpga_cw310_mask_rom"],
         "bitstream_gcp": ["@bitstreams//:bitstream_mask_rom"],
         "//conditions:default": ["@bitstreams//:bitstream_mask_rom"],
     }),
 )
 
-# TODO(lowRISC/opentitan#13603): Use `select` once we're uploading these
-# artifacts to GCP.
 filegroup(
-    name = "mask_rom_otp_dev",
-    srcs = [
-        "//hw/bitstream/vivado:fpga_cw310_mask_rom_otp_dev",
-    ],
+    name = "rom_mmi",
+    srcs = select({
+        "bitstream_skip": ["skip.bit"],
+        "bitstream_vivado": ["//hw/bitstream/vivado:rom_mmi"],
+        "bitstream_gcp": ["@bitstreams//:rom_mmi"],
+        "//conditions:default": ["@bitstreams//:rom_mmi"],
+    }),
+)
+
+filegroup(
+    name = "otp_mmi",
+    srcs = select({
+        "bitstream_skip": ["skip.bit"],
+        "bitstream_vivado": ["//hw/bitstream/vivado:otp_mmi"],
+        "bitstream_gcp": ["@bitstreams//:otp_mmi"],
+        "//conditions:default": ["@bitstreams//:otp_mmi"],
+    }),
+)
+
+bitstream_splice(
+    name = "fpga_cw310_mask_rom",
+    src = ":test_rom",
+    data = "//sw/device/silicon_creator/mask_rom:mask_rom_fpga_cw310_scr_vmem",
+    meminfo = ":rom_mmi",
+    tags = ["manual"],
+)
+
+bitstream_splice(
+    name = "fpga_cw310_mask_rom_otp_dev",
+    src = ":mask_rom",
+    data = "//hw/ip/otp_ctrl/data:img_dev",
+    meminfo = ":otp_mmi",
+    tags = ["manual"],
```

Comment:
This rule, OTOH, can always just be a target of various tests, as the behavior of the test has more to do with the parameters set in the OTP instead of the selection of the particular bitstream source.  The `src` should probably use the `select`ing `filegroup` so that `--define bitstream={gcp,vivado,...}` uses the requested bitstream as the source of the splice.

Reply:
I'm actually just going to omit this OTP target for now. I've been working with @jon-flatley and we do think it's worth defining OTP images in Bazel, but details are still a bit hazy. (I'm intending to write up our rationale this week.)